? c/src/lib/libcpu/powerpc/mpc6xx/mmu/mmu.c
? c/src/lib/libcpu/powerpc/mpc6xx/mmu/mmu.h
? cpukit/libmmu
? testsuites/libtests/mmutests
Index: cpukit/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/Makefile.am,v
retrieving revision 1.86
diff -u -r1.86 Makefile.am
--- cpukit/Makefile.am	5 May 2009 12:52:29 -0000	1.86
+++ cpukit/Makefile.am	4 Jul 2009 17:10:07 -0000
@@ -12,6 +12,7 @@
 SUBDIRS += libcsupport libblock libfs
 SUBDIRS += libnetworking librpc
 SUBDIRS += libi2c
+SUBDIRS += libmmu
 SUBDIRS += libmisc
 SUBDIRS += libmd
 SUBDIRS += libgnat
@@ -135,6 +136,10 @@
 ## i2c
 include_rtems_HEADERS += libi2c/libi2c.h
 
+
+## mmu
+include_rtems_HEADERS += libmmu/libmmu.h
+
 ## serdbg
 if LIBSERDBG
 include_rtems_HEADERS += libmisc/serdbg/serdbgcnf.h libmisc/serdbg/serdbg.h \
Index: cpukit/configure.ac
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/configure.ac,v
retrieving revision 1.162
diff -u -r1.162 configure.ac
--- cpukit/configure.ac	15 Apr 2009 08:36:13 -0000	1.162
+++ cpukit/configure.ac	4 Jul 2009 17:10:07 -0000
@@ -364,6 +364,7 @@
 libmisc/Makefile
 libi2c/Makefile
 libmd/Makefile
+libmmu/Makefile
 zlib/Makefile
 httpd/Makefile
 ftpd/Makefile
Index: cpukit/preinstall.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/preinstall.am,v
retrieving revision 1.37
diff -u -r1.37 preinstall.am
--- cpukit/preinstall.am	5 May 2009 12:52:29 -0000	1.37
+++ cpukit/preinstall.am	4 Jul 2009 17:10:07 -0000
@@ -255,6 +255,10 @@
 	$(INSTALL_DATA) $< $(PROJECT_INCLUDE)/rtems/libi2c.h
 PREINSTALL_FILES += $(PROJECT_INCLUDE)/rtems/libi2c.h
 
+$(PROJECT_INCLUDE)/rtems/libmmu.h: libmmu/libmmu.h $(PROJECT_INCLUDE)/rtems/$(dirstamp)
+	$(INSTALL_DATA) $< $(PROJECT_INCLUDE)/rtems/libmmu.h
+PREINSTALL_FILES += $(PROJECT_INCLUDE)/rtems/libmmu.h
+
 if LIBSERDBG
 $(PROJECT_INCLUDE)/rtems/serdbgcnf.h: libmisc/serdbg/serdbgcnf.h $(PROJECT_INCLUDE)/rtems/$(dirstamp)
 	$(INSTALL_DATA) $< $(PROJECT_INCLUDE)/rtems/serdbgcnf.h
Index: cpukit/wrapup/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/wrapup/Makefile.am,v
retrieving revision 1.66
diff -u -r1.66 Makefile.am
--- cpukit/wrapup/Makefile.am	5 May 2009 13:36:07 -0000	1.66
+++ cpukit/wrapup/Makefile.am	4 Jul 2009 17:10:07 -0000
@@ -43,6 +43,9 @@
 TMP_LIBS += ../libmisc/libstackchk.a
 TMP_LIBS += ../libmisc/libcpuuse.a
 
+TMP_LIBS += ../libmmu/libmmu.a
+
+
 ## XXX temporarily removed because it causes a
 ## XXX number of BSPs to not link "main(){}" used by autoconf
 # if LIBSERDBG
Index: testsuites/libtests/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/libtests/Makefile.am,v
retrieving revision 1.14
diff -u -r1.14 Makefile.am
--- testsuites/libtests/Makefile.am	7 May 2009 17:48:15 -0000	1.14
+++ testsuites/libtests/Makefile.am	4 Jul 2009 17:10:07 -0000
@@ -5,7 +5,7 @@
 ACLOCAL_AMFLAGS = -I ../aclocal
 
 SUBDIRS = cpuuse malloctest heapwalk putenvtest monitor monitor02 rtmonuse \
-    stackchk termios rtems++ tztest
+    stackchk termios rtems++ tztest mmutests
 
 include $(top_srcdir)/../automake/subdirs.am
 include $(top_srcdir)/../automake/local.am
Index: testsuites/libtests/configure.ac
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/libtests/configure.ac,v
retrieving revision 1.21
diff -u -r1.21 configure.ac
--- testsuites/libtests/configure.ac	7 May 2009 17:48:15 -0000	1.21
+++ testsuites/libtests/configure.ac	4 Jul 2009 17:10:07 -0000
@@ -44,5 +44,6 @@
 stackchk/Makefile
 termios/Makefile
 tztest/Makefile
+mmutests/Makefile
 ])
 AC_OUTPUT
Index: testsuites/samples/hello/init.c
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/samples/hello/init.c,v
retrieving revision 1.15
diff -u -r1.15 init.c
--- testsuites/samples/hello/init.c	11 Jan 2008 22:50:10 -0000	1.15
+++ testsuites/samples/hello/init.c	4 Jul 2009 17:10:07 -0000
@@ -1,24 +1,13 @@
-/*  Init
- *
- *  This routine is the initialization task for this test program.
- *  It is called from init_exec and has the responsibility for creating
- *  and starting the tasks that make up the test.  If the time of day
- *  clock is required for the test, it should also be set to a known
- *  value by this function.
- *
- *  Input parameters:  NONE
- *
- *  Output parameters:  NONE
- *
- *  COPYRIGHT (c) 1989-1999.
- *  On-Line Applications Research Corporation (OAR).
- *
- *  The license and distribution terms for this file may be
- *  found in the file LICENSE in this distribution or at
- *  http://www.rtems.com/license/LICENSE.
- *
- *  $Id: init.c,v 1.15 2008/01/11 22:50:10 joel Exp $
- */
+/*
+*  COPYRIGHT (c) 1989-2009.
+*  On-Line Applications Research Corporation (OAR).
+*
+*  The license and distribution terms for this file may be
+*  found in the file LICENSE in this distribution or at
+*  http://www.rtems.com/license/LICENSE.
+*
+*  $Id$
+*/
 
 #define CONFIGURE_INIT
 #include "system.h"
